src/main.c

```diff
@@ -229,15 +229,13 @@ init_signals(void)
 }
 
 /**@internal
- * Main execution loop
+ * Setup from Configuration values
  */
 static void
-main_loop(void)
+setup_from_config(void)
```

Comment:
Maybe we should pass the config variable here as an argument.
I do not have a strong opinion here.